tests/integration/test_bolt_driver.py

```diff
@@ -167,3 +168,20 @@ def test_test_multi_db_specify_database(bolt_uri, auth):
     except ClientError as error:
         # FAILURE {'code': 'Neo.ClientError.Database.DatabaseNotFound' - This message is sent from the server
         assert error.args[0] == "Database does not exist. Database name: 'test_database'."
+
+
+def test_bolt_driver_fetch_size_config(bolt_uri, auth):
+    # python -m pytest tests/integration/test_bolt_driver.py-s -v -k test_bolt_driver_fetch_size_config
```

Comment:
Pls clean up all unnecessary comments in this PR. The comments cannot be any code or command. It could only be human-readable text to describe the code if the code cannot self-explain itself.